src/FNA3D_Driver_OpenGL.c

```diff
@@ -358,7 +364,9 @@ static int32_t XNAToGL_TextureInternalFormat[] =
 	GL_RG16F,				/* SurfaceFormat.HalfVector2 */
 	GL_RGBA16F,				/* SurfaceFormat.HalfVector4 */
 	GL_RGBA16F,				/* SurfaceFormat.HdrBlendable */
-	GL_RGBA8				/* SurfaceFormat.ColorBgraEXT */
+	GL_RGBA8,				/* SurfaceFormat.ColorBgraEXT */
+	GL_SRGB_ALPHA_EXT,			/* SurfaceFormat.ColorSrgbEXT */
+	GL_COMPRESSED_RGBA_S3TC_DXT5_EXT,	/* SurfaceFormat.Dxt5SrgbEXT */
```

Comment:
Looking around this file, I'm wondering if this doesn't work because it's no different from the regular DXT5 values? Is there supposed to be a DXT5 SRGB enum somewhere?